### DISCUSSION TITLE
Translate a phrase without propagation to Other Occurrences

### DISCUSSION THREAD

**Discussion by Heshmatkhah** [WeblateOrg/weblate#10003] [Q&A]
Hi
I have a word that should be translated,
Based on the context it has different meanings.
If I translate one, All of them will change, is there a way to disable this for the phrase or select other occurrences that should be updated (have the same meaning and context)?

**Comment by nijel** [comment]
In case the strings are not supposed to be propagated, disable this in component configuration: https://docs.weblate.org/en/latest/admin/projects.html#allow-translation-propagation

**Comment by Heshmatkhah** [comment]
I want this project-wide but be turned off for specific Phrases.
Also, when I turn `Allow translation propagation` off in my component, it still replace other components translations.

**Reply by nijel** [reply]
`Allow translation propagation` toggles whether strings can be propagated to this component. There is currently no more fine-grained control, if you have some suggestions on that topic, please follow up at https://github.com/WeblateOrg/weblate/issues/3166
